src/black/linegen.py

```diff
@@ -919,10 +931,20 @@ def _maybe_split_omitting_optional_parens(
             yield result
 
 
-def _prefer_split_rhs_oop(rhs_oop: RHSResult, mode: Mode) -> bool:
+def _prefer_split_rhs_oop_over_rhs(
+    rhs_oop: RHSResult, rhs: RHSResult, mode: Mode
+) -> bool:
     """
-    Returns whether we should prefer the result from a split omitting optional parens.
+    Returns whether we should prefer the result from a split omitting optional parens
+    (rhs_oop) over the original (rhs).
     """
+    # If we have multiple targets, we prefer more `=`s on the head vs pushing them to
+    # the body
+    lhs_equal_count = [leaf.type for leaf in rhs.head.leaves].count(token.EQUAL)
```

Comment:
```suggestion
    rhs_equal_count = [leaf.type for leaf in rhs.head.leaves].count(token.EQUAL)
```
Not sure where the "lhs" comes from here?

Reply:
The thought process was, that rhs.head becomes the lhs in the cases where we have multiple assignments. But yeah, better to stick with `rhs_head`. 👍